libarchive/archive_read_support_format_tar.c

```diff
@@ -2935,6 +2962,13 @@ header_gnutar(struct archive_read *a, struct tar *tar,
 		}
 	}
 
+	/* Grab fields common to all tar variants. */
```

Comment:
We used to parse `header_common` at the beginning of `header_gnutar`, but moved it to the end for consistency with `header_ustar`.